relay-general/src/protocol/session.rs

```diff
@@ -151,6 +182,9 @@ pub struct SessionUpdate {
     /// The session event attributes.
     #[serde(rename = "attrs")]
     pub attributes: SessionAttributes,
+    /// The abnormal mechanism.
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub abnormal_mechanism: Option<AbnormalMechanism>,
```

Comment:
Looking at the `abnormal_mechanism()` helper method, there seems to be no semantic difference between the two `None`s, so I think we could simplify this to:

```suggestion
    #[serde(default, skip_serializing_if = "AbnormalMechanism::is_none")]
    pub abnormal_mechanism: AbnormalMechanism,
```

Reply:
Addressed in https://github.com/getsentry/relay/pull/1665/commits/5c5a504b8fa4a5966909cf2fd341c22cca315e4b